Bottom: ea5ee6942bd1f6abc3b29f1c33587219770b18e1
Top:    21d32fa82638c9302635a493f86c99690c86ea0a
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-18 21:46:19 +0000

hid/gtk: Fix broken pad-center finding code in the netlist window

Looks like I made a copy+paste error in the code which shows the user
the location of a particular netlist connection.

(Caught with clang)


---

diff --git a/src/hid/gtk/gui-netlist-window.c b/src/hid/gtk/gui-netlist-window.c
index 35b5427..f87491d 100644
--- a/src/hid/gtk/gui-netlist-window.c
+++ b/src/hid/gtk/gui-netlist-window.c
@@ -295,7 +295,7 @@ node_selection_changed_cb (GtkTreeSelection * selection, gpointer data)
         {
           PadTypePtr pad = (PadTypePtr) conn.ptr2;
           x = pad->Point1.X + (pad->Point2.X - pad->Point1.X) / 2;
-          y = pad->Point1.Y + (pad->Point2.Y - pad->Point2.Y) / 2;
+          y = pad->Point1.Y + (pad->Point2.Y - pad->Point1.Y) / 2;
           gui->set_crosshair (x, y, 0);
           ghid_lead_user_to_location (x, y);
           break;
